.gitignore

```diff
@@ -60,4 +60,18 @@ local.properties
 
 
 ## VSCODE ***
-/.vscode/
\ No newline at end of file
+/.vscode/
+
+##############################
+## Workflow Service SDK Generated files
+##############################
+
+workflow-service-sdk/src/
+workflow-service-sdk/api/
+workflow-service-sdk/docs/
+workflow-service-sdk/*gradle*
+workflow-service-sdk/.openapi-generator/
+workflow-service-sdk/build.*
+workflow-service-sdk/api-docs.json
+workflow-service-sdk/git_push.sh
+workflow-service-sdk/.travis.yml
```

Comment:
Shouldn't those lines be added to the `.gitignore` file inside the module, if it already exists?